docs/telemetry.md

```diff
@@ -1,55 +1,273 @@
+    {
+      "id": "B5BDC47B-B717-4EF5-AEDF-41A17C9C18BB",
+      "createTime": "2023-09-01T10:56:49Z",
+      "instanceId": "B5BDC47B-B717-4EF5-AEDF-41A17C9C18BA",
+      "productFamily": "PRODUCT_FAMILY_PS",
+      "metrics": [
+        {
+          "key": "OS",
+          "value": "Ubuntu"
+        },
+        {
+          "key": "pillar_version",
+          "value": "7.0.0"
+        }
+      ]
+    }
+  ]
+}
 ```
 
+The agent sends information about the database and metrics.
+
+| Key | Description |
+|---|---|
+| "id" | A generated Universally Unique Identifier (UUID) version 4 |
+| "createTime" | UNIX timestamp |
+| "instanceId" | The DB Host ID. The value can be taken from the `instanceId`, the `/usr/local/percona/telemetry_uuid` or generated as a UUID version 4 if the file is absent. |
+| "productFamily" | The value from the file path |
+| "metrics" | An array of key:value pairs collected from the Metrics file.
+
+The following operating system-level metrics are sent with each check:
+
+| Key | Description |
+|---|---|
+| "OS" | The name of the operating system |
+| "hardware_arch" | The type of process used in the environment |
+| "deployment" | How the application was deployed. <br> The possible values could be "PACKAGE" or "DOCKER". |
+| "installed_packages" | A list of the installed Percona packages.|
+
+The information includes the following:
+
+* Package name
+
+* Package version - the same format as Red Hat Enterprise Linux or Debian
+
+* Package repository - if possible
+
+The package names must fit the following pattern:
+
+* `percona-*`
+
+* `Percona-*`
+
+* `proxysql*`
+
+* `pmm`
+
+* `etcd*`
+
+* `haproxy`
+
+* `patroni`
+
+* `pg*`
+
+* `postgis`
+
+* `wal2json`
+
 ## Disable telemetry
 
-Starting with Percona Server for MongoDB 7.0.4-2, telemetry is enabled by default. If you decide not to send usage data to Percona, you can set the `PERCONA_TELEMETRY_DISABLE=1` environment variable for either the root user or in the operating system prior to the installation process.
+Percona software you just installed collects information about how you use it (telemetry) by default. Here's how to turn it off:
 
-=== "Debian-derived distribution"
+### Within the first 24 hours
 
-    Add the environment variable before the install process.
+This is the easiest option. You have 24 hours to disable telemetry before any information is collected or sent.
 
-    ```{.bash data-prompt="$"}
-    $ sudo PERCONA_TELEMETRY_DISABLE=1 apt install percona-server-mongodb
+Use the following command to turn it off:
+
+```{.bash data-prompt=$}
+$ sudo systemctl stop percona-telemetry-agent
+```
+
+Even after stopping the telemetry agent service, a different part of the software (Telemetry subsystem) continues to create the Metrics file related to telemetry every day and saves that file for seven days.
+
+### To disable the Telemetry subsystem
+
+You can disable the Telemetry subsystem by running the `setParameter` command in one of the following ways:
```

Comment:
This above sentence is ambiguous. It depends on what you mean by "running the setParameter command".
By literary running the `setParamater` command you can disable Telemetry subsystem **only temporarily** until server restart.
Consider changing to:
```
You can disable the Telemetry subsystem temporarily or persistently by setting the `perconaTelemetry` server parameter in one of the following ways:
```

Reply:
Is there a way to disable Telemetry permanently?